persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -121,3 +110,9 @@ UPDATE Conversation SET mls_last_keying_material_update= ? WHERE mls_group_id =
 
 selectByKeyingMaterialUpdate:
 SELECT mls_group_id FROM Conversation WHERE mls_group_state = ? AND protocol = ? AND mls_last_keying_material_update - ? <0 AND mls_group_id IS NOT NULL;
+
+isUserMember:
+SELECT user FROM Member WHERE conversation = ? AND user = ?;
+
+whoDeletedMeInConversation:
+SELECT sender_user_id FROM Message WHERE id = (SELECT message_id FROM MessageMemberChangeContent WHERE conversation_id = ? AND member_change_type = "REMOVED" AND member_change_list LIKE :self_user_id LIMIT 1);
```

Comment:
I already showed you my suggestion, maybe it would be good to know others opinion as well, because I think we need to take into account that there may be multiple messages like that so we need to choose the last one

```
SELECT sender_user_id
FROM Message
WHERE id IN 
    (
    SELECT message_id
    FROM MessageMemberChangeContent
    WHERE conversation_id = :conversation_id
    AND member_change_type = "REMOVED"
    AND member_change_list LIKE :self_user_id
    )
ORDER BY Datetime(date) DESC
LIMIT 1;
```